### ISSUE TITLE
[Mod Support]  Applied Energistics 2: Trousers Edition

### ISSUE THREAD

**Issue by Exaxxion** [F32Organization/Faithful32-1.12.2#225]
**Mod Name**
Applied Energistics 2: Trousers Edition

**Link to the mod (ideally Curseforge)**
https://github.com/PrototypeTrousers/Applied-Energistics-2/releases

Proto's fork of AE2 fixes a lot of performance problems and adds many new excellent quality-of-life features. We are including it by default as an approved off-CurseForge mod in forthcoming releases of Omnifactory. F32 is my texture pack of choice and one we recommend as supporting all of the mods we use.

There are a few new textures that were added to `resources\main\assets\appliedenergistics2\textures\guis\states.png` but because F32 overrides the texture mappings for AE2, these textures do not appear:

![image](https://user-images.githubusercontent.com/11907282/113641774-26008980-9633-11eb-8f79-b72177a8f9be.png)

![image](https://user-images.githubusercontent.com/11907282/113641790-2d279780-9633-11eb-9f38-1c94c5ba5db8.png)

**Comment by Evolution0** [comment]
Pretty sure this would require an optional resource zip as the directories are named the same. Resource packs are "dumb" in the sense that its just a zip with a directory structure inside that overrides that which is normally contained in the Assets directory of the mods jar.

I don't see why a F32 Addon resource pack can't be made though, Its been a while since I've made textures for the pack but I could sure give it a shot.

**Comment by Exaxxion** [comment]
Thank you for investigating this.

PAE2 is better optimized than base AE2 for playing with channels disabled, as that is the default setting in Omnifactory (though to my knowledge it can still be used fine with them enabled). That could be why you were not seeing a controller block in your environment (or if you launched Omnifactory, definitely why).

As I understand it, PAE2's textures are a superset of the original mod's so it should be possible to safely use the PAE2 atlas for either mod.

**Comment by Evolution0** [comment]
Yeah I grabbed the latest public version of the pack and threw PAE2 in it to test the textures.

Not sure why I originally thought it would be an issue (Maybe I've been out of the game so long lol), you're right the mainline mod doesn't use that part of the texture atlas so simply won't display them, I'll add a PR for it so it can be moved into the resource pack.
